src/res/flame.html

```diff
@@ -66,6 +83,12 @@ <h1>/*title:*/</h1>
 		return '#' + (p[0] + ((p[1] * v) << 16 | (p[2] * v) << 8 | (p[3] * v))).toString(16);
 	}
 
+	const legendDetail = document.getElementById('legendDetail');
+	const legendItems = legendDetail.getElementsByTagName('div');
+	for(let i = 0; i < palette.length; i++) {
+		legendItems[i].style.backgroundColor = getColor(palette[i]);
```

Comment:
`getColor` returns random color within palette range. Should we use stable color for the legend?

Reply:
Yes. Colors will not match exactly to what is in the flamegraph in any case, I don't mind either way.